Source/Extensions/Blazorise.DataGrid/EventArguments/DataGridSortColumnInfo.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace Blazorise.DataGrid;
+
+/// <summary>
+/// Holds the sort order and sort direction information of a specific column
+/// </summary>
+public class DataGridSortColumnInfo
```

Comment:
The class should be moved under the `./Models` subfolder instead of `./EventArguments`.

Reply:
Agreed, good catch.

Reply:
There are currently three files in /Models:
- DataGridColumnInfo, namespace `Blazorise.DataGrid`
- DataGridRowInfo, namespace `Blazorise.DataGrid.Models`
- DataGridSortColumnInfo, namespace `Blazorise.DataGrid`

Is it intentional that the three files have partially different namespaces? If not - what would be the desired namespace?

Reply:
`DataGridRowInfo` namespace is wrong. we just keep everything under the base namespace, generally at least.